packages/israel/README.md

```diff
@@ -0,0 +1,35 @@
+# 🇮🇳 Israel
```

Comment:
Could you also add this country in the main `README.md` please? I forgot to write it in the contribution guide 🙇